crates/miden-objects/src/batch/note_tree.rs

```diff
@@ -1,7 +1,11 @@
+use alloc::vec::Vec;
+
 use miden_crypto::{
     hash::rpo::RpoDigest,
     merkle::{MerkleError, SimpleSmt},
 };
```

Comment:
Not from this PR, but these could also come from the `crate` section.

Reply:
Just created https://github.com/0xPolygonMiden/miden-base/issues/1145 to tackle this.